### ISSUE TITLE
Traffic Management and Tracing page data have no data in multi-cluster project

### ISSUE THREAD

**Issue by wenxin-01** [kubesphere/kubesphere#4818]
**Describe the Bug**
![image](https://user-images.githubusercontent.com/68640256/164684066-2e7482a4-ee21-4e36-8610-eac34e12b49f.png)
![image](https://user-images.githubusercontent.com/68640256/164684454-8f2169d9-5d04-48af-afdd-3fbe47d0d1cc.png)

![image](https://user-images.githubusercontent.com/68640256/164683765-72139fc2-9783-4544-9111-bf61334ab762.png)
![image](https://user-images.githubusercontent.com/68640256/164683883-057d183a-f346-477b-bdf0-4a68136bb45d.png)

**Versions Used**
KubeSphere:   `v3.3.0-alpha.1`

**How To Reproduce**
1、In a multi-cluster project, enable the Nodeport mode gateway
2、Create a composing application and set the application route
3、Access Application Route
4、viewing Traffic Management and tracking page
5、the graph display on the Traffic Management page and The tracking page have no data

/priority high
/assign @zryfish

**Comment by zryfish** [comment]
Request parameter rateInterval is `undefined`, which I believe is a mistake. 
```
/kapis/clusters/host/servicemesh.kubesphere.io/v1alpha2/namespaces/multi-pro/health?rateInterval=undefineds&type=app
```

There is a duration select control in single cluster application project while under mulit-cluster project, there isn't. Could this be the cause? @harrisonliu5 

![Screenshot 2022-04-24 at 15 45 37](https://user-images.githubusercontent.com/3326354/164965922-3135cc05-acc3-4230-af79-e1b068b9c4a3.png)

**Comment by wenxin-01** [comment]
issue fix in `v3.3.0-alpha.2`
![image](https://user-images.githubusercontent.com/68640256/167233346-75e1cdd8-1e65-4836-a306-c8d496423822.png)
